app/src/hvr/cpp/native-lib.cpp

```diff
@@ -175,6 +175,8 @@ JNI_METHOD(void, nativeOnSurfaceChanged)
   }
 
    sAppContext->mDevice->EnterVR(*sAppContext->mEgl);
+   // We need the session to be created before we can update the environment
+   BrowserWorld::Instance().UpdateEnvironment();
```

Comment:
I know that we often say that noapi and wavevr and not really maintained, but for simple cases it makes sense to update also the other native-libs. Would you mind adding them to these other backends?

Reply:
Done!